controllers/secret_controller.go

```diff
@@ -177,7 +177,7 @@ func (r *SecretReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		for _, node := range nodes.Items {
 			// Only clear the annotation for Nodes associated with Machines, as the clearing of the annotation is used
 			// solely to kick off the deletion and recreation of Machines.
-			if _, present := node.Annotations[BYOHAnnotation]; present {
+			if _, present := node.GetLabels()[BYOHLabel]; present {
```

Comment:
I suggest updating the filter when you are getting the list of nodes.

Reply:
I did not want to do this as we need both BYOH and Machine nodes for https://github.com/openshift/windows-machine-config-operator/pull/508